buildkite/scripts/connect-to-berkeley.sh

```diff
@@ -30,9 +30,17 @@ apt-get install --allow-downgrades -y "mina-${TESTNET_NAME}=${MINA_DEB_VERSION}"
 # Remove lockfile if present
 rm ~/.mina-config/.mina-lock ||:
 
+mkdir -p /root/libp2p-keys/
+# Pre-generated random password for this quick test
+export MINA_LIBP2P_PASS=eithohShieshichoh8uaJ5iefo1reiRudaekohG7AeCeib4XuneDet2uGhu7lahf
```

Comment:
not super familiar with this tool but it looks like you're setting the password to the key before you're even generating the key?  how does that work?